packages/features/shell/Shell.tsx

```diff
@@ -188,6 +190,32 @@ function useRedirectToOnboardingIfNeeded() {
   };
 }
 
+function useRedirectTo2FAIfNeeded() {
```

Comment:
hook to redirect organiztion user to two-factor-auth web page whenever they are navigating within the app